js-miniapp-sample/src/pages/landing.js

```diff
@@ -13,6 +13,16 @@ type LandingProps = {
   getHostInfo: Function,
 };
 
+window.addEventListener('miniAppWebviewClosed', function (e) {
+  let message = e.detail;
+  console.log(message);
+});
+
+window.addEventListener('miniAppOnPause', function (e) {
```

Comment:
I think we also need a `miniappresume` event. I know that we have `miniAppWebviewClosed` but that's specifically for use when the webview was closed. There are other cases where the Host App could pause the mini app when they wouldn't get a `miniAppWebviewClosed` - i.e. putting the app in the background, receive a phone call, etc.